src/core/sal/panda/PandaController.cc

```diff
@@ -30,12 +30,12 @@ PandaController::PandaController()
 
 PandaController::~PandaController()
 {
-	delete m_Mem;
 	std::vector<ConcreteCPU*>::iterator it = m_CPUs.begin();
 	while (it != m_CPUs.end()) {
 		delete *it;
 		it = m_CPUs.erase(it);
 	}
+	delete &getMemoryManager();
```

Comment:
At this point only the MemoryManager for MEMTYPE_RAM would be deleted, since this is the default for getMemoryManager().

Reply:
This is fine for Panda, since there is only one kind of memor manager.